django/applications/catmaid/control/treenodeexport.py

```diff
@@ -264,7 +266,7 @@ def create_path(self, connector_link):
         relation_folder =  self.relid_to_rel_folder[connector_link.relation_id]
 
         # Create path output_path/neuron_id/relation_name/connector_id
-        connector_path = os.path.join(self.output_path, neuron_folder,
+        connector_path = os.path.join(self.output_path, neuron_folder, # type: ignore # mypy cannot prove that self.output_path will always have a value
```

Comment:
I guess this method could test for this initially and raise an error if `self.output` hasn't been set.

Reply:
That's fair. I'm also okay with just documenting it. Mainly don't want a lot of type:ignore declarations with no notes (both for someone looking at it later and because it's a lazy habit to turn off type safety without a reason).